p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -2175,6 +2178,28 @@ struct Parsed_headers {
 }
 ~ End P4Example
 
+### Tuple types { #sec-tuple-types }
```

Comment:
When I detangled conflicts, I couldn't tell if this was meant to be deleted. So I kept it. Can you confirm @mbudiu-vmw